### ISSUE TITLE
Disable page morphing

### ISSUE THREAD

**Issue by warpech** [PuppetJs/PuppetJs#42]
It should be possible to turn off page morphing/link capturing in Puppets.

Add a control for it to https://github.com/Polyjuice/starcounter-debug-aid once it is done.

This is point 11 from https://github.com/Starcounter/Starcounter/issues/2468

**Comment by tomalec** [comment]
Now morphing is available only for [PuppetDOM](https://github.com/PuppetJs/PuppetJs#puppetdom) - PuppetJS extension for browser integration, and can be switched via `listen()`/`unlisten()` methods.

If you do not need morphing at all, consider use of pure `PuppetJs` (w/o `PuppetDOM`) - library for syncing JS objects via network (w/o DOM binding/listening overhead)

**Comment by miyconst** [comment]
@warpech should I add a checkbox to `starcounter-debug-aid` to `PuppetDOM` `listen/unlisten`?

**Comment by warpech** [comment]
Yes, totally!

**Comment by miyconst** [comment]
Done, however disabling morphing breaks apps which use `redirectUrl`. Puppet-dom not only listens to mouse click, but also to history change event.

When disabled, `juicy-redirect` changes url, but not loads new page.

**Comment by warpech** [comment]
Great problem to have! I created a separate issue for it

**Comment by warpech** [comment]
I think this is solved. Problems that result from page morphing being disabled (such as #67) should be tracked in separate issues.
